stripe.go

```diff
@@ -279,6 +284,34 @@ type BackendImplementation struct {
 	requestMetricsBuffer chan requestMetrics
 }
 
+type unmarshalTargeter interface {
```

Comment:
Why do we need a new interface? Can't we implement the telemetry callback using the LastResponseSetter interface?

Reply:
Tackled in [custom UnmarshalJSON instead of unmarshaltargeter](https://github.com/stripe/stripe-go/pull/1779/commits/e3566667d74928a8685dabb3e5e3aa34d68fdbb5)